test/integration/include-ts/a.ts

```diff
@@ -0,0 +1,19 @@
+// test 1: sub-module
+import Vue from '/web_modules/vue/dist/vue.esm.browser.js';
+
+// test 2: default export
+import VueRouter from '/web_modules/vue-router.js';
+
+Vue.use(VueRouter);
+
+// test 3: type import
+
+import type { Component } from "vue"
```

Comment:
Type import test

Reply:
FANTASTIC! Glad we have this under test too, since I doubt es-module-lexer is specifically testing for it (or even considers this in scope).

Reply:
It’s a ~bug~ Feature™